FirstProject/assets/common.less

```diff
@@ -0,0 +1,18 @@
+.button {
+  border: none;
```

Comment:
General comments(fix in ALL files):
1. Use variables for defining colors and do not hard code them(eg define them in a colors.less)
2. Separate structure and skin